examples/oneview_logical_interconnect_group.yml

```diff
@@ -13,38 +13,113 @@
 ---
 - hosts: all
   vars:
     config: "{{ playbook_dir }}/oneview_config.json"
     # Inform the permitted Interconnect Type URI to configure Interconnect Map
-    permitted_interconnect_type_name: 'Virtual Connect SE 32Gb FC Module for Synergy'
+    permitted_interconnect_type_name: 'Virtual Connect SE 40Gb F8 Module for Synergy'
+    permitted_interconnect_type_name_1: 'Synergy 20Gb Interconnect Link Module'
     scope_uris:
       - '/rest/scopes/d0324c4b-56c9-485b-b7dd-8f6218a4b5bb'
       - '/rest/scopes/9422ce05-dc39-4358-b723-c8c825eae412'
   tasks:
+ # Below task creates a Logical Interconnect Group with Interconnects, upLinkPorts, uplinkSets containing EthernetNetworks
     - name: Create a Logical Interconnect Group
       oneview_logical_interconnect_group:
         config: "{{ config }}"
         state: present
         data:
           name: 'Test Logical Interconnect Group'
-          uplinkSets: []
           enclosureType: 'SY12000'
+          uplinkSets: 
+            - name: 'e23 uplink set'
+              mode: 'Auto'
+              networkType: 'Ethernet'
+              ethernetNetworkType: 'Tagged'
+              networkUris:
+                - '/rest/ethernet-networks/221905e8-4e22-4888-9ab8-50a2d4ebaac2'
+              logicalPortConfigInfos:
+               - desiredSpeed: 'Auto'
+                 logicalLocation:
+                      locationEntries:
+                        - relativeValue: 1
+                          type: 'Enclosure'
+                        - relativeValue: 61
+                          type: 'Port'
+                        - relativeValue: 3
+                          type: 'Bay'
+            - name: 'e23 uplink set1'
+              mode: 'Auto'
+              networkType: 'Ethernet'
+              ethernetNetworkType: 'Untagged'
+              networkUris:
+                - '/rest/ethernet-networks/1866ce7d-db93-4ecf-a888-546a18cd64cd'
+              logicalPortConfigInfos:
+               - desiredSpeed: 'Auto'
+                 logicalLocation:
+                      locationEntries:
+                        - relativeValue: 2
+                          type: 'Enclosure'
+                        - relativeValue: 62
+                          type: 'Port'
+                        - relativeValue: 6
+                          type: 'Bay'
           interconnectBaySet: 3    
-          enclosureIndexes: [1, 2]
+          enclosureIndexes: [1,2,3]
           redundancyType: "HighlyAvailable"
           interconnectMapTemplate:
             interconnectMapEntryTemplates:
-              - logicalDownlinkUri: ~
+              - enclosureIndex: 3
+                logicalLocation:
+                    locationEntries:
+                        - relativeValue: 3
+                          type: "Enclosure"
+                        - relativeValue: 6
+                          type: "Bay"
+                permittedInterconnectTypeUri: "/rest/interconnect-types/c16e12d7-06ea-4bbf-ad22-f46d78540dcc"
```

Comment:
try to provide permittedInterconnectTypeName instead of permittedInterconnectTypeUri  in example and check if it works. Also add a comment for this playbook mentioning for which enclosure setup will this yaml works. (This playbook works for synergy 3 enclosure setup)

Reply:
done